com.prateekmedia.appimagepool.yaml

```diff
@@ -0,0 +1,50 @@
+app-id: com.prateekmedia.appimagepool
+runtime: org.gnome.Platform
+runtime-version: "40"
+sdk: org.gnome.Sdk
+command: appimagepool
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --share=network
+  - --device=all
+  - --filesystem=home
+  - --filesystem=xdg-download
+  - --talk-name=org.freedesktop.Notifications
+  - --own-name=com.prateekmedia.appimagepool
+modules:
+  - name: appimagepool
+    buildsystem: simple
+    only-arches:
+      - x86_64
+    build-commands:
+      - ls appimagepool
+      - install -Dm644 appimagepool/appimagepool /app/appimagepool/appimagepool
+      - cp -R appimagepool/data /app/appimagepool/data
+      - cp -R appimagepool/lib /app/appimagepool/lib
+      - chmod +x /app/appimagepool/appimagepool
+      - ls /app
+      - chmod +x /app/appimagepool
+      - install -Dm644 appimagepool/appimagepool.svg /app/share/icons/hicolor/scalable/apps/appimagepool.svg
+      - mkdir /app/bin
+      - ln -s /app/appimagepool/appimagepool /app/bin/appimagepool
+      - install -Dm644 appimagepool/appimagepool.desktop /app/share/applications/appimagepool.desktop
+      - install -Dm644 com.prateekmedia.appimagepool.appdata.xml /app/share/metainfo/com.prateekmedia.appimagepool.appdata.xml
+    sources:
+      - type: archive
+        only-arches:
+          - x86_64
+        url: https://github.com/prateekmedia/appimagepool/releases/download/2.3.0/appimagepool-x86_64.tar.gz
```

Comment:
It happens that this is not a gzip file but just a tar file, so flatpak-builder is confused. (and fails)